### ISSUE TITLE
🐛 HMR/watching not working with 1.5 on Linux

### ISSUE THREAD

**Issue by kasbah** [parcel-bundler/parcel#669]
HMR doesn't seem to watch for file changes with 1.5 (or built from master as of 1cd3cf703a80f753386682866534dc96d9a38c6a) on Ubuntu Linux 16.04 (64-bit). It works fine when downgrading to 1.4.1. 

### Configuration

Just running the most basic example with an  index.html and index.js. 

```js
{
  "devDependencies": {
    "parcel-bundler": "1.5"
  }
}

```

### Possible Solution

Maybe a regression in Chokidar but I tried downgrading Chokidar to 1.6.1 and 1.5 and upgrading to 2.0.0 and it didn't help. 

### Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 1.5 and master
| Node             | v8.9.4
| npm/Yarn         |5.6.0/1.3.2
| Operating System | Ubuntu 16.04 (64-bit)

**Comment by DeMoorJasper** [comment]
This is probably an issue with safewrite

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by davidnagli** [comment]
This has to do with an IDE feature called safe write which get’s in the way of the file-watcher. We tried to get around this using fs polling enabled via a CLI flag (see #238), but this was a hack, and would still require user knowledge of safe-write so instead we ended up deciding to to add instructions to the docs showing the user how to properly disable this in their IDE (see parcel-bundler/website#53). 

Nobody implemented the documentation changes discussed in that issue yet, but it’s definitely something that we should do.

The main tracking issue for this is #221
